<ui version="4.0" >
 <class>KoDocumentInfoAboutWidget</class>
 <widget class="QWidget" name="KoDocumentInfoAboutWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>546</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="filePathInfoLabel" >
     <property name="text" >
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLabel" name="filePathLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblTitleDesc" >
     <property name="text" >
      <string>Title:</string>
     </property>
     <property name="buddy" >
      <cstring>leTitle</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLineEdit" name="leTitle" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="lblSubjectDesc" >
     <property name="text" >
      <string>Subject:</string>
     </property>
     <property name="buddy" >
      <cstring>leSubject</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QLineEdit" name="leSubject" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="lblKeywordsDesc" >
     <property name="text" >
      <string>Keywords:</string>
     </property>
     <property name="buddy" >
      <cstring>leKeywords</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QLineEdit" name="leKeywords" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="lbCommentsDesc" >
     <property name="text" >
      <string>Comments:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy" >
      <cstring>meComments</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2" >
    <widget class="QTextEdit" name="meComments" />
   </item>
   <item row="6" column="0" colspan="3" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="lblTypeDesc" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QLabel" name="lblType" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="lblCreatedDesc" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Created:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QLabel" name="lblCreated" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="QLabel" name="lblModifiedDesc" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Modified:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" >
    <widget class="QLabel" name="lblModified" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0" >
    <widget class="QLabel" name="lblLastPrintedDesc" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Last printed:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" >
    <widget class="QLabel" name="lblLastPrinted" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0" >
    <widget class="QLabel" name="lblEditingDesc" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Total editing time:</string>
     </property>
     <property name="buddy" >
      <cstring>pbReset</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1" >
    <widget class="QLabel" name="lblEditing" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="2" >
    <widget class="QPushButton" name="pbReset" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>&amp;Reset</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" >
    <widget class="QLabel" name="lblRevisionDesc" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Revision number:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" >
    <widget class="QLabel" name="lblRevision" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="0" >
    <widget class="QLabel" name="lblEncryptedDesc" >
     <property name="text" >
      <string>Encryption:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblEncryptedPic" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblEncrypted" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="13" column="2" >
    <widget class="QPushButton" name="pbEncrypt" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Encrypt</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <includes/>
 <resources/>
 <connections/>
</ui>
